docs/content/components/overlay/ContextualHelp/contextualhelp.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: ContextualHelp
+caption: Displays contextual help via popover on button click
+badge: new
+---
+
+The `<ContextualHelp>` component provides inline assistance to help users understand specific parts of your interface. It displays a small button with an icon that, when clicked, opens a popover with explanatory content. This content can be rich and interactive, unlike tooltips which are intended for brief, passive messages.
+
+Use `<ContextualHelp>` when users might need deeper guidance or context, especially for complex settings, form inputs, or unfamiliar actions.
+
+## Anatomy
+
+<Image
+  src="/contextualhelp/contextualhelp-anatomy.jpg"
+  alt="Anatomy of tooltip"
+  width={800}
+  height={550}
+  className="mx-auto block"
+/>
+
+- **Trigger button:** A small icon button that activates the help popover.
+- **Popover:** A floating container for detailed help content.
+- **Dialog content:** Custom content area that may contain headings, paragraphs, or even interactive elements.
+
+## Appearance
+
+<AppearanceDemo component={title} />
+
+## Usage
+
+ContextualHelp is ideal when brief tooltips are insufficient and users might benefit from more elaborate information. Avoid overusing it—too many help popovers can clutter the UI. Aim to use it selectively on parts of your app where extra guidance is truly needed.
```

Comment:
for me still not 100% sure what the difference between tooltip and contextualHelp is. 🙈